.github/workflows/lint.yml

```diff
@@ -0,0 +1,53 @@
+name: "Lint"
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  pull_request:
+    branches: [master]
```

Comment:
Even if it's accepted, it's not so YAML compliant.
It should be something like:

```yaml
on:
  pull_request:
    branches:
      - master
[...]
```